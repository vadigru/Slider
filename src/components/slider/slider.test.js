import React from "react";
import renderer from "react-test-renderer";

import Slider from "./slider.jsx";

it(`Should render Slider component`, () => {
  const tree = renderer.create(
      <Slider
        demoMode={false}
        width={1200}
        height={600}
        infinite={true}
        caption={true}
        autoplay={true}
        autoplayDelay={5000}
        indicators={true}
        arrows={true}
        adaptiveSlides={true}
        animatedSwipe={true}
        animationTime={350}
        slidesCount={3}
      >
        <div>
          <h2>slide 1</h2>
        </div>
      </Slider>
  );

  expect(tree).toMatchSnapshot();
});
