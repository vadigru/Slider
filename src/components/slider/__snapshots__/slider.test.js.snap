// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Slider component 1`] = `
<div
  className="slider-wrapper"
>
  
  <section
    className="slider"
    onClickCapture={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    style={
      Object {
        "height": 600,
        "maxWidth": 1024,
        "width": "100%",
      }
    }
  >
    <div
      className="slides"
      onAnimationEnd={[Function]}
      onMouseDown={[Function]}
      onMouseMove={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "animationDuration": "350ms",
          "animationName": "",
          "animationTimingFunction": "ease",
          "left": "-100%",
        }
      }
    >
      <div
        className="slide
                    
                    "
        style={
          Object {
            "backgroundImage": "url()",
            "minWidth": "50%",
          }
        }
      >
        <h2>
          slide 1
        </h2>
      </div>
      <div
        className="slide
                    
                    "
        style={
          Object {
            "backgroundImage": "url()",
            "minWidth": "50%",
          }
        }
      >
        <h2>
          slide 1
        </h2>
      </div>
      <div
        className="slide
                    slide-1
                    "
        style={
          Object {
            "backgroundImage": "url()",
            "minWidth": "50%",
          }
        }
      >
        <h2>
          slide 1
        </h2>
      </div>
    </div>
    
    
  </section>
</div>
`;
